symtabAPI/src/Object-elf.C

```diff
@@ -52,6 +52,8 @@
 
 #include "Object-elf.h"
 
+#include <regex>
```

Comment:
If regex is no longer used, consider remove this?

Reply:
Ok, removed